awx/ui/client/test/unit/side-nav-item.spec.js

```diff
@@ -0,0 +1,62 @@
+describe('Components | Side Nav Item', () => {
+    let $compile;
+    let $rootScope;
+    let element;
+    let scope;
+
+    beforeEach(() => {
+        angular.mock.module('gettext');
+        angular.mock.module('I18N');
+        angular.mock.module('ui.router');
+        angular.mock.module('at.lib.services')
+        angular.mock.module('at.lib.components')
+    });
+
+    beforeEach(angular.mock.inject((_$compile_, _$rootScope_) => {
+        $compile = _$compile_;
+        $rootScope = _$rootScope_;
+        scope = $rootScope.$new();
+
+        element = angular.element('<at-layout><at-side-nav><at-side-nav-item icon-class="fa-tachometer" route="dashboard" name="DASHBOARD"></at-side-nav-item></at-layout></at-side-nav><');
```

Comment:
the last `...<at-side-nav><` here might be a typo

Reply:
Ah. Thanks for catching that.